packages/doenetml-iframe/src/iframe-editor-index.ts

```diff
@@ -21,6 +21,7 @@ document.addEventListener("DOMContentLoaded", () => {
         undefined,
         {
             ...doenetEditorProps,
+            keyboardIsOutsideIframe: true,
```

Comment:
This isn't really about whether the keyboard is outside the iframe, as whether the built-in keyboard should be suppressed. Maybe `externalVirtualKeyboardProvided`?

Reply:
True, though we do then assume that messages come in from `window.parent`. Not sure if that exists other than with an iframe. I'm happy to rename the prop, though.